docs/README.md

```diff
@@ -141,7 +140,7 @@ You can also obtain the image and review our tagging scheme on [DockerHub](https
 
 ### Building Agent Binary on Linux
 
-The agent requires `v1.42+` of rustc [cargo-make](https://github.com/sagiegurari/cargo-make) to build. If the proper versions of rustc and cargo are installed; then simply run the following command to build the agent:
+The agent requires rustc [cargo-make](https://github.com/sagiegurari/cargo-make) to build. If rustc and cargo are installed; then simply run the following command to build the agent:
```

Comment:
re: installed
i would link this to rustup https://www.rust-lang.org/tools/install.

Reply:
Updated.